src/webpack.config.ts

```diff
@@ -95,31 +95,31 @@ function webpackConfig(args: Partial<BuildArgs>) {
 		}, args => {
 			return {
 				'src/main': [
-					path.join(basePath, 'src/main.css'),
+					path.join(basePath, 'src/main.m.css'),
```

Comment:
Updated to not be a `.m.css` file!  It does make sense that we won't want the "main" css file to be a css module.